Makefile

```diff
@@ -200,8 +200,7 @@ ol3cesium: ol .build-artefacts/ol3-cesium
 	git show; \
 	ln -T -f -s ../../../../ol3-cesium-plugin/ src/plugins/geoadmin; \
 	( cd cesium; [ -f node_modules/.bin/gulp ] || npm install ); \
-	( cd cesium; if [ -f "Build/Cesium/Cesium.js" ] ; then echo 'Skipping Cesium minified build'; else node_modules/.bin/gulp minifyRelease; fi ); \
-	( cd cesium; if [ -f "Build/CesiumUnminified/Cesium.js" ] ; then echo 'Skipping Cesium debug build'; else node_modules/.bin/gulp combine; fi ); \
+	( cd cesium; if [ -e "Build/Cesium" ] && [ -e "Build/CesiumUnminified" ]; then echo 'Skipping Cesium debug build'; else npm run combine; rm -rf Build/Cesium; npm run minifyRelease; fi ); \
```

Comment:
I see.
The drawback of your PR is that we now always compile Cesium twice.
When developping, I often removed manually the combined flavour of Cesium to only pay one compilation.
There may be a trick, by prepending another target first, like this (not tested):
`node_modules/.bin/gulp generateDocumentation combine`.

We can try to optimize later, so +1 for merging.